### PR TITLE
Show profile links only by hovering on photos

### LINKED ISSUE #306: Show Profile links only by hovering on the photo
### Expected results
Profile links will be visible only when one hovers on a Mentor/student/contributor's photo.
Please go through [https://codeheat.org/](https://codeheat.org/)

### Actual results
At present, Profile links are always visible

### Screenshot of the issue
**Actual**
image
**Expected**
image

### PR DESCRIPTION
**Pull request by shikhar-scs**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

This fixes #306 

new1

new2

Social Media links on the main page were made to appear on hover rather than staying there always.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,243 @@
+GEM
+  remote: https://rubygems.org/
+  specs:
```
Comment (abishekvashok) [inline Gemfile.lock]: Don't include gem.lock

**Review by abishekvashok — CHANGES_REQUESTED** [review]

File: Gemfile.lock (hunk 1 above)
Comment (shikhar-scs) [reply Gemfile.lock]: @abishekvashok  Removed it in the next commit

File: _includes/header.html [hunk 2]
```diff
@@ -5,7 +5,9 @@
 		<title>FOSSASIA | Google Code-In 2017</title>
 
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <link href="css/font-awesome.min.css" rel="stylesheet" type="text/css" media="all">
+		<!--<link href="css/font-awesome.min.css" rel="stylesheet" type="text/css" media="all">-->
+		<link href="css/font-awesome.css" rel="stylesheet" type="text/css" media="all">
+		<link href="css/font-awesome.css/font-awesome.min.css" rel="stylesheet" type="text/css" media="all">
```
Comment (abishekvashok) [inline _includes/header.html]: Please properly align the code, and pls use the min version only.

**Review by abishekvashok — CHANGES_REQUESTED** [review]

File: _includes/header.html (hunk 2 above)
Comment (shikhar-scs) [reply _includes/header.html]: @abishekvashok  done

**Comment by abishekvashok** [comment]
@shikhar-scs can you squash your commits via a rebase.

**Comment by shikhar-scs** [comment]
@abishekvashok  I'm running into problems because of that. Could you squash and merge please

**Comment by ParthS007** [comment]
@shikhar-scs Please provide a preview link

**Comment by shikhar-scs** [comment]
@ParthS007 ok ill do that ASAP on heroku

**Review by isuruAb — COMMENTED** [review]
Please provide a preview link

**Comment by shikhar-scs** [comment]
@abishekvashok Actually I've only worked on nodeJS till now and thus deploying to heroku in ruby is taking a lot of time with different errors. Currently there is no ProcFile I guess. Can you help me with that plz

**Comment by anshumanv** [comment]
@shikhar-scs please use simple gh-pages for providing a preview link - https://pages.github.com/

**Comment by shikhar-scs** [comment]
@anshumanv That was seriously a life saver (I wont be able to thank you enough). Here is the preview link 
https://shikhar-scs.github.io/gci17.fossasia.org/
Please review @ParthS007 @isuruAb @abishekvashok

**Comment by ParthS007** [comment]
well done @shikhar-scs . can you please apply hovering effect to all the photos and increase the size of profile icons (change Github icon also) as it will look uniform and completely resolves #306

**Comment by shikhar-scs** [comment]
@ParthS007  Yeah Sure I'll Do that and commit tomorrow

**Comment by ParthS007** [comment]
@shikhar-scs Use Icons just similar to what is used in https://2016.fossasia.org/

**Comment by anshumanv** [comment]
Glad to help you @shikhar-scs :smile: 
Please mirror the behavior you see on https://fossasia.org/

**Comment by shikhar-scs** [comment]
Hey @ParthS007 I've made the website uniform in terms of hoverable display of social media links. Also I have fixed the github icon too (using the one from font awesome). Also, font-awesome.min.css doesnt seem to work i guess ( i spent half an hour correcting the github icon but it always showed a broken rectangular box icon ) so i have included a cdn for that too.
I hope this completely resolves #306

**Comment by ParthS007** [comment]
@shikhar-scs Have you updated the link ?

**Comment by ParthS007** [comment]
BTW you have done a great job 💯

**Comment by anshumanv** [comment]
You can scale up the icons a bit btw. :smile:

**Review by anshumanv — APPROVED** [review]
LGTM!

**Comment by shikhar-scs** [comment]
@ParthS007 Yes the preview link shows things as it should. [here](https://shikhar-scs.github.io/gci17.fossasia.org/) it is .

Thanx a lot everyone

**Comment by anshumanv** [comment]
Can you please confirm if it works on students and contributors section as well?

**Comment by shikhar-scs** [comment]
Yes it does. That was primarily why I made this latest commit

**Comment by anshumanv** [comment]
Yes, Awesome @shikhar-scs, just saw.
I think it took a while for gh-pages to update the changes. :+1:

**Comment by anshumanv** [comment]
Hey @shikhar-scs this is almost good to go, as the last step can you add a last commit to increase the icon size as well please?

Thanks, good work!

**Comment by shikhar-scs** [comment]
@anshumanv Done that. Plz have a look. Also tell me if  i need to scale them down a bit

**Comment by ParthS007** [comment]
@shikhar-scs Just keep the size same as https://fossasia.org/ and your work is ready to merge 🎉

**Comment by anshumanv** [comment]
Thanks for the changes but please mirror the ones on the org website. :smile:

**Comment by shikhar-scs** [comment]
@ParthS007 Its optimum i guess you can anyways check it [here](https://shikhar-scs.github.io/gci17.fossasia.org/)

should i make them smaller?

**Comment by shikhar-scs** [comment]
@ParthS007 @anshumanv any more changes?

**Comment by ParthS007** [comment]
@shikhar-scs I am seeing different size of icons in different place.
image
image

**Comment by ParthS007** [comment]
The second one is correct size

**Comment by shikhar-scs** [comment]
@ParthS007 Sorry for that error. And now i have lowered down all of them . Please have a look. Its reflecting in gh-pages too.

**Review by ParthS007 — APPROVED** [review]

**Comment by shikhar-scs** [comment]
Thanx a lot @ParthS007 @anshumanv for merging this.

**Comment by ParthS007** [comment]
That's amazing @shikhar-scs  , Keep it up and learn to squash commits

**Comment by shikhar-scs** [comment]
Yup sure Ill do that the next time

**Comment by ParthS007** [comment]
@shikhar-scs Have a look at #332

**Comment by shikhar-scs** [comment]
@ParthS007 I noticed something odd this morning. Have a look at the main site - the contributors list.
mmm

and now my gh-pages site -the contributors list.
nnn

why do the icons on the main site still appear below?

**Pull request merged**